worldedit-core/src/main/java/com/sk89q/worldedit/command/ToolUtilCommands.java

```diff
@@ -169,9 +169,14 @@ public void size(
     public void traceMask(
             Player player, LocalSession session,
             @Arg(desc = "The trace mask to set", def = "")
-                    Mask maskOpt
+            Mask maskOpt
     ) throws WorldEditException {
-        session.getBrushTool(player, false).setTraceMask(maskOpt);
+        BrushTool brushTool = session.getBrushTool(player, false);
+        if (brushTool == null) {
+            player.printError(Caption.of("worldedit.brush.none.equipped"));
```

Comment:
```suggestion
            player.print(Caption.of("worldedit.brush.none.equipped"));
```
printError doesn't listen to color codes in the lang file.